main.go

```diff
@@ -56,11 +73,13 @@ func main() {
 		}
 	}
 
-	if findArg(args, flDryRun) != -1 {
```

Comment:
do you need to delete `findArg` (+its tests) since you're not using anymore?

Reply:
`ldflags.go` still uses it.